### PR TITLE
chore: get eval metrics from ingest in

### PR DESCRIPTION
**Pull request by Klaijan**

### CODE REVIEW

File: test_unstructured_ingest/check-diff-evaluation-metrics.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@ trap cleanup EXIT
 # to update ingest test fixtures, run scripts/ingest-test-fixtures-update.sh on x86_64
 if [ "$OVERWRITE_FIXTURES" != "false" ]; then
     # force copy (overwrite) files from metrics-tmp (new eval metrics) to metrics (old eval metrics)
-    cp -rf "$EXPECTED_OUTPUT_DIR" "$EXPECTED_OUTPUT"
+    cp -rf "$OUTPUT_DIR" "$EXPECTED_OUTPUT_DIR"
```
Comment (cragwolfe) [inline test_unstructured_ingest/check-diff-evaluation-metrics.sh:32]: these vars could be more clear: maybe `EXPECTED_OUTPUT_DIR` -> `METRICS_DIR`  AND `OUTPUT_DIR` -> `TMP_METRICS_LATEST_RUN_DIR`?

Comment (Klaijan) [reply test_unstructured_ingest/check-diff-evaluation-metrics.sh]: Renamed!

File: unstructured/ingest/evaluate.py [hunk 2]
```diff
@@ -52,6 +52,7 @@ def measure_text_edit_distance_command(
     export_dir: str,
     weights: Tuple[int, int, int],
 ):
+    click.echo(export_dir)
```
Comment (cragwolfe) [inline unstructured/ingest/evaluate.py:55]: can remove this now?

Comment (Klaijan) [reply unstructured/ingest/evaluate.py:55]: Ah yes, I missed that one!

**Review by cragwolfe — APPROVED** [review]

**Pull request merged**